docassemble/HousingCodeChecklist/data/questions/housing_code_interview.yml

```
@@ -236,6 +236,10 @@ question: |
 subquestion: |
   Need help to get repairs or stop your landlord from harassing you? Pick a task.
   
+  <div class="alert alert-danger h-20 mh-20" role="alert">
+    <p class="h6 alert-danger"><i class="fa-solid fa-triangle-exclamation fa-xl"></i> In order to use UpToCode your rental unit must be located in Massachusetts.</p>
+  </div>
```

```suggestion
  Need help to get repairs or stop your **Massachusetts** landlord from harassing you? Pick a task.
  
```

Just thinking to make this warning clear but not as aggressive.